bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog for Bookmarking a message."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, jump_url: str = None, *args) -> None:
+        """Bookmarks a message."""
+        if jump_url is not None and jump_url != "*":
+            if "discordapp.com/channels" not in jump_url:
+                await ctx.send("I can't find the associated message,"
+                               " the command support the following syntax:\n"
+                               " `.bm [message url] (bookmark name)`")
+                return
+        if jump_url is None or jump_url == "*":
+            channel = ctx.channel
+            async for x in channel.history(limit=2):
+                message_id = x.id
+            try:
+                message = await channel.fetch_message(message_id)
+            except discord.NotFound:
+                await ctx.send(">>> Message not in this"
+                               " channel\n"
+                               "Or\n"
+                               "You are trying to Pin"
+                               " your own message,\n"
+                               "Please use the channel"
+                               " where message is.")

```

Comment:
No.

1. We use K&R indentation style.
```python
await ctx.send(
    "Message goes here. "
    "Additional content goes like this."
)
```

2. All spaces should be at the end of strings, not in the beginning of the string.
3. There's no capital P in pin.
4. Why are there so many newlines? why on earth would `Or` be on a line entirely of its own? Currently this would print a message like
```
>>> Message not in this channel
Or
You are trying to Pin your own message,
Please use the channel where message is.
```

5. Why is there a capital P after the comma? We don't capitalize words after commas.
6. Why are we using `>>>` for this? It's a weird way to style it.
7. Do we even need this error? Can you explain when and why this would ever be displayed?

Reply:
>>> set the format like this
![image](https://user-images.githubusercontent.com/45149585/70222555-3cad7c80-1770-11ea-82ec-584b6ceeaeed.png)
7. no one knows the future , if we decide to add a exception of not pinning your own message 
this can be used .
I simplified the message now it now prints
`Message not in this channel
Or You are trying to pin your own message,
Please use the channel where message is.`

Reply:
Why are you using a quote block? An embed will looks way better, in the same style than the others error message

Reply:
`7. Removing`
its dead now